vernac/vernacentries.ml

```diff
@@ -2238,6 +2244,11 @@ let interp ?proof ~atts ~st c : Proof_global.t option =
   (* This one is possible to handle here *)
   | VernacAbort id    -> CErrors.user_err  (str "Abort cannot be used through the Load command")
 
+  (* Loading a file requires access to the control interpreter *)
+  | VernacLoad (verbosely,fname) ->
```

Comment:
The comment was to explain the assert false so isn't relevant anymore.

Reply:
thanks